docs/en-US/commands/Find-JiraFilter.md

```diff
@@ -0,0 +1,298 @@
+    [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Searches for filters. This operation is similar to Get filters except that the results can be refined to include filters that have specific attributes. For example, filters with a particular name. When multiple attributes are specified only filters matching all attributes are returned.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Find-JiraFilter -Name 'ABC'
+```
+
+This example finds all JIRA filters that include ABC in the name.  The search is case insensitive.
+
+### EXAMPLE 2
+
+```powershell
+Find-JiraFilter -Name """George Jetsons Filter"""
+```
+
+This example finds a JIRA filter by exact name (case insensitive)
+
+### EXAMPLE 3
+
+```powershell
+'My','Your' | Find-JiraFilter
+```
+
+This example demonstrates use of the pipeline to search for multiple filter Name(s).  The search is case insensitive.
+
+### EXAMPLE 4
+
+```powershell
+Find-JiraFilter -Name 'My','Your'
+```
+
+This example demonstrates the use of a list of names to search for multiple filter Name(s).  The search is case insensitive.
+
+### EXAMPLE 5
+
+```powershell
+Find-JiraFilter -AccountId 'owner' -First 3 -Skip 3
+```
+
+This example finds all JIRA filters belonging to a specific owner, and illustrates the use of the -First and -Skip Paging parameters.
+
+### EXAMPLE 6
+
+```powershell
+Find-JiraFilter -Project 10315 -First 8
+```
+This example finds all JIRA filters belonging to a specific project.
+
+Use Get-JiraProject to retrieve a numeric Project.
+
+### Example 7
+
+```powershell
+Find-JiraFilter -Name """George Jetsons Filter""" -Fields 'description','jql'
+```
+
+This example finds the JIRA filter named "George Jetsons Filter" but only expands the fields listed.
+
+By retrieving only the data really needed, the payload the server sends is
+reduced, which speeds up the search.
+
+## PARAMETERS
+
+### -Name
+
+String used to perform a case-insensitive partial match with name.  An exact match can be requested by including quotes (refer to the examples above).
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -AccountId
+
+User account ID used to return filters with the matching owner.accountId.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -GroupName
+
+Group name used to return filters that are shared with a group that matches sharePermissions.group.groupname.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Project
+
+Project object, key, or id used to return filters that are shared with a project that matches sharePermissions.project.id.
```

Comment:
Please update this to match the `JiraPS.Project` object

Reply:
Done.